pillar/nginx/chalkradio_redirect.sls

```diff
@@ -0,0 +1,34 @@
+{% set tlds = ['com', 'org', 'net'] %}
+
+nginx:
+    servers:
+    managed:
+      {% for tld in tlds %}
+      chalkradio-{{ tld }}-redirect:
+        enabled: True
+        config:
+          - server:
+              - server_name: chalkradiopodcast.{{ tld }}
```

Comment:
Rather than defining a separate server block for each domain you can just have all of the domains in a space separated list in the `server_name` parameter.

Reply:
Oh, didn't realize that. will change ...